backend/otr/db/4_init.sql

```diff
@@ -168,3 +180,23 @@ SELECT
 FROM region
 WHERE region_id % 7 = 0
 ON CONFLICT DO NOTHING;
+
+-- set random meeting dates qualifications
+UPDATE qualification
+SET meeting_date_id = (
+    SELECT md.meeting_date_id FROM meeting_date md
+    WHERE md.date < CURRENT_DATE
+    ORDER BY random() + qualification_id LIMIT 1
+) WHERE 1=1;
+
+-- set qualification term begin and end dates
+UPDATE qualification
+SET begin_date = (
+    SELECT md.date FROM meeting_date md
+    WHERE md.meeting_date_id = qualification.meeting_date_id
+),
+end_date = (
+    SELECT md.date FROM meeting_date md
+    WHERE md.meeting_date_id = qualification.meeting_date_id
+) + '5 years'::interval
```

Comment:
Päättymisaika voisi olla ehkä 5 vuoden sijaan vuoden tai 1,5 vuotta alkamisajankohdasta. Tällöin meillä olisi testidatassa myös umpeutuneita pätevyyksiä.